docs/sources/next/javascript-api/k6-net-grpc/stream/stream-on.md

```diff
@@ -10,10 +10,10 @@ aliases:
 
 Set up handler functions for various events on the gRPC stream.
 
-| Parameter | Type     | Description                                  |
-| --------- | -------- | -------------------------------------------- |
-| event     | string   | The event name to define a handler for.      |
-| handler   | function | The function to call when the event happens. |
+| Parameter | Type     | Description                                                                                                                                                                                             |
+| --------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| event     | string   | The event name to define a handler for.                                                                                                                                                                 |
+| handler   | function | The function to call when the event happens. The second argument of the handler is the message's [`Metadata`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-net-grpc/stream/message-metadata). |
```

Comment:
I think it will be better if this and the other mention of handler point to some page with more information about the first *and* second argument they take. As I don't think the current documentation mentions how many arguments int can take and what they are.

Reply:
I was thinking about that, so yeah since it's a second call I'm going to apply it

Reply:
Created an issue here so we don't forget about it: https://github.com/grafana/k6-docs/issues/1745